windows_containers/windows-containers-release-notes-8-x.adoc

```diff
@@ -0,0 +1,59 @@
+:_content-type: ASSEMBLY
+[id="windows-containers-release-notes-8-x"]
+= {productwinc} release notes
+include::_attributes/common-attributes.adoc[]
+:context: windows-containers-release-notes
+
+toc::[]
+
+[id="about-windows-containers"]
+== About {productwinc}
+
+Windows Container Support for Red Hat OpenShift enables running Windows compute nodes in an {product-title} cluster. Running Windows workloads is possible by using the Red Hat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. With Windows nodes available, you can run Windows container workloads in {product-title}.
+
+The release notes for Red Hat OpenShift for Windows Containers tracks the development of the WMCO, which provides all Windows container workload capabilities in {product-title}.
+
+ifndef::openshift-origin[]
+[id="getting-support"]
+== Getting support
+
+// wording taken and modified from https://access.redhat.com/support/policy/updates/openshift#windows
+
+Windows Container Support for Red Hat OpenShift is provided and available as an optional, installable component. Windows Container Support for Red Hat OpenShift is not part of the {product-title} subscription. It requires an additional Red Hat subscription and is supported according to the link:https://access.redhat.com/support/offerings/production/soc/[Scope of coverage] and link:https://access.redhat.com/support/offerings/production/sla[Service level agreements].
+
+You must have this separate subscription to receive support for Windows Container Support for Red Hat OpenShift. Without this additional Red Hat subscription, deploying Windows container workloads in production clusters is not supported. You can request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
+
+For more information, see the Red Hat OpenShift Container Platform Life Cycle Policy document for link:https://access.redhat.com/support/policy/updates/openshift#windows[{productwinc}].
+
+If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
+endif::openshift-origin[]
+
+[id="wmco-8-0-0"]
+== Release notes for Red Hat Windows Machine Config Operator 8.0.0
+
+This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 8.0.0 were released in link:https://access.redhat.com/errata/RHSA-2023:1372[RHSA-2023:1372].
```

Comment:
I tried accessing the [errata link](https://access.redhat.com/errata/RHSA-2023) but I got a 404 , is that ok at this point in time?

Reply:
no problem, I just wanted to confirm if that was the case. Thanks for the quick answer.